routers/common/middleware.go

```diff
@@ -50,6 +51,28 @@ func Middlewares() []func(http.Handler) http.Handler {
 
 	handlers = append(handlers, middleware.StripSlashes)
 
+	handlers = append(handlers, func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(resp http.ResponseWriter, req *http.Request) {
+			var path string
+			rctx := chi.RouteContext(req.Context())
+			if rctx != nil && rctx.RoutePath != "" {
+				path = rctx.RoutePath
+			} else {
+				path = req.URL.Path
+			}
+			fmt.Println(strings.HasPrefix(path, "//"))
+			if len(path) > 1 && strings.HasPrefix(path, "//") {
```

Comment:
Suffix?

Reply:
Suffix is already being trimmed in `routers/common/middleware.go:52` (`middleware.StripSlashes`). Here is prefix trim (`//path/to`)